src/app/modules/item/pages/chapter-user-progress/chapter-user-progress.component.html

```diff
@@ -47,22 +47,49 @@
           <td *ngFor="let col of columns">
             <ng-container [ngSwitch]="col.field">
               <ng-container *ngSwitchCase="'title'">
-                <a
-                  class="alg-link"
-                  [ngClass]="{'disabled': rowData.id === itemData?.item?.id}"
-                  (click)="onClick(rowData)"
-                >
-                  {{ rowData.title }}
-                </a>
+                <div class="title">
+                  <a
+                    class="alg-link"
+                    [ngClass]="{'disabled': rowData.id === itemData?.item?.id}"
+                    (click)="onClick(rowData)"
+                  >
+                    {{ rowData.title }}
+                  </a>
+
+                  <ng-container *ngIf="rowData.type === 'Task' && rowData.currentUserPermissions">
+                    <button
+                      pButton
+                      class="alg-button p-button-rounded p-small-button p-button-outlined load-answer-button"
+                      [label]="(!!(watchedGroup$ | async) ? 'watchingGroup' : '') | i18nSelect : {
+                        watchingGroup: 'View the best answer',
+                        other: 'Reload the best answer'
+                      }"
+                      (click)="onClick(rowData)"
+                      *ngIf="rowData.currentUserPermissions | allowsWatchingAnswers"
```

Comment:
That's not the only case: if you are not observing, so checking your own solution, you can always reload it.

Reply:
Fixed